code/game/objects/items/devices/flashlight.dm

```diff
@@ -363,7 +363,11 @@
 	. = ..()
 	if(.)
 		user.visible_message("<span class='notice'>[user] cracks and shakes [src].</span>", "<span class='notice'>You crack and shake [src], turning it on!</span>")
-		START_PROCESSING(SSobj, src)
+		activate()
+
+/obj/item/device/flashlight/glowstick/proc/activate()
+	on = TRUE
+	START_PROCESSING(SSobj, src)

```

Comment:
I couldn't get the glowsticks to activate remotely otherwise, though there may be another solution